src/main/java/longah/node/Transaction.java

```diff
@@ -69,8 +82,26 @@ public void parseTransaction(String expression, MemberList members) throws LongA
         // Check for existence of all parties involved in the transaction in the group.
         String lenderName = splitInput[0].trim();
         this.lender = members.getMember(lenderName);
+        String borrowNameAmount;
         for (int i = 1; i < splitInput.length; i++) {
-            String borrowNameAmount = splitInput[i].trim();
+
+            if (i == splitInput.length - 1) {
+                String[] inputWithDateTime = splitInput[i].split("t/");
+                if (inputWithDateTime.length == 2) {
+                    try {
+                        this.transactionTime = LocalDateTime.parse(inputWithDateTime[1].trim(),
+                                DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm"));
```

Comment:
Sorry im a dd-mm-yyyy supremecist..

Reply:
Out of all the programming languages you chose to speak facts

Reply:
Resolved!